ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__acm-ocp4.15-lp-interop.yaml

```diff
@@ -46,18 +46,18 @@ tests:
         [
           {"step": "ipi-conf*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
           {"step": "ipi-install*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
-          {"step": "ipi-deprovision*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
+          {"step": "ipi-deprovision*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 3}, "jira_additional_labels": ["4.15-lp","self-managed-lp","acm-lp"]},
```

Comment:
I'm thinking we should leave ipi-deprovision as a priority 2.

My thinking is that in a case where a cluster comes up, then we see both the must-gather step and deprovision step fail then only a ticket will be created for the failure in must-gather and we'll get no notification that we failed to deprovision the cluster.

I think deprovision should always be a priority 2 with cluster provisioning being a priority 1. So the only time that we should ignore a deprovisioning failure is when the cluster install fails.

What are your thoughts?